lib/internal/cluster/round_robin_handle.js

```diff
@@ -13,7 +15,7 @@ function RoundRobinHandle(key, address, port, addressType, fd, flags) {
   this.free = [];
   this.handles = [];
   this.handle = null;
-  this.server = net.createServer(assert.fail);
+  this.server = net.createServer(assertFail);
```

Comment:
Perhaps I misunderstood Brian's suggestion but this is identical to `net.createServer(assert)`, isn't it? The no-args invocation is equivalent to `assert(undefined)`, which in turn coerces to `false`.

(Nit-picking^2, I realize that. :-))

Reply:
> Perhaps I misunderstood Brian's suggestion but this is identical to `net.createServer(assert)`, isn't it?

No because with that, the assertion will always pass because the argument provided to `assert()` will be the socket, which will always coerce to truthy.

This will instead get `assert(false, socket)` so it will always result in an `AssertionError`.

Reply:
I'm wondering if it makes sense to add a low-overhead `assert.fail()`as it will be more clear all the way around and not that much overhead--just another two or three lines in the module.